homeassistant/components/airzone/entity.py

```diff
@@ -130,6 +136,11 @@ def __init__(
         )
         self._attr_unique_id = entry.unique_id or entry.entry_id
```

Comment:
`AirzoneSystemEntity` and `AirzoneZoneEntity` almost look like they share enough to have a common base class